fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1837,6 +1837,25 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_refresh_hive_partitions_statistics = true;
 
+    /**
+     * cache BaseTable, there is no harm of consistence, so default is true.
+     * we set it as a configuration to avoid that unexpected case.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_iceberg_meta_cache = true;
```

Comment:
It's better to use as catalog properties

Reply:
will do it.